src/Utility/Text.php

```diff
@@ -1120,4 +1133,25 @@ public static function slug($string, $options = [])
 
         return $string;
     }
+
+    /**
+     * Get default html tags not counted for truncate.
+     *
+     * @return array Html tags.
+     */
+    public static function getHtmlNoCount()
+    {
+        return static::$_defaultHtmlNoCount;
+    }
+
+    /**
+     * Set html tags not counted for truncate.
+     *
+     * @param array $htmlNoCount New html tags not counted for truncate.
+     * @return void
+     */
+    public static function setHtmlNoCount($htmlNoCount)
+    {
```

Comment:
Can we have a typehint here to ensure the parameter is an array?

Reply:
I need this into 3.5.x, i must remove methods for this time ?

Reply:
Yes. I don't think we need to make these tags configurable.